src/time-slot-bar-week/time-slot-bar-week-days.tsx

```diff
@@ -152,27 +156,31 @@ export const TimeSlotBarWeekDays = ({
     const generateStyleProps = (day: Dayjs) => {
         const dateStartWithYear = day.format(dateFormat);
         const disabled = isDisabled(day);
-
-        const styleCircleProps: StyleProps = {},
-            styleLabelProps: StyleProps = {};
+        const dayCellStyleProps: CellStyleProps = {};
+        const isHoverEnabled = enableSelection && !disabled;
 
         if (disabled) {
-            styleCircleProps.$disabledDisplay = true;
-            styleLabelProps.$disabledDisplay = true;
+            dayCellStyleProps.disabled = true;
         }
 
-        styleCircleProps.$interactive = enableSelection && !disabled;
+        dayCellStyleProps.interactive = !enableSelection
+            ? null
+            : isHoverEnabled;
+
+        if (isHoverEnabled && day === hoverDay) {
```

Comment:
to use the built in comparator here
```suggestion
        if (isHoverEnabled && hoverDay && day.isSame(hoverDay, "day")) {
```